### ISSUE TITLE
How to set threshold when considering IsoMax+ as the loss function

### ISSUE THREAD

**Issue by e96031413** [dlmacedo/entropic-out-of-distribution-detection#1]
Thanks for your great work,

I am wondering how do you set threshold to detect the given sample is "in-distribution or out-of-distribution"?

**Comment by dlmacedo** [comment]
In practice, you may set the threshold of two ways.

You may collect some random out-of-distribution samples and use also your train/validation set to define the threshold. This way, you may have a estimation of the performance you may expect in the field.

Other option is only using you train or validation set without access to a random out-of-distribution. In this case, you may set a threshold by making, for example, 95% of your in-distribution data to be considered indeed in-distribution using the chosen threshold. In this case, however, you will not have a estimation of the expect performance.

**Comment by dlmacedo** [comment]
> Thanks for your great work,
> 
> I am wondering how do you set threshold to detect the given sample is "in-distribution or out-of-distribution"?

I hope I have answered your question.

**Comment by dlmacedo** [comment]
> Thanks for your great work,
> 
> I am wondering how do you set threshold to detect the given sample is "in-distribution or out-of-distribution"?

I only notice now.

This is why I am only answering now.

**Comment by e96031413** [comment]
Thanks for your detaild answer.
